src/Tracking.php

```diff
@@ -566,4 +566,61 @@ private static function filter_add_to_cart_attributes( array $args, WC_Product $
 
 		return $args;
 	}
+
+
+	/**
+	 * Get the formatted warning message for the potential conflicting tags.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string The warning message.
+	 */
+	public static function get_third_party_tags_warning_message() {
+
+		$third_party_tags = self::get_third_party_installed_tags();
+
+		if ( empty( $third_party_tags ) ) {
+			return '';
+		}
+
+		return wp_kses_post(
+			sprintf(
+				/* Translators: %1$s Conflicting plugins */
+				__( 'The following installed plugin(s) can potentially cause problems with tracking: %1$s. %2$sRemove conflicting plugins%3$s or %4$smanage tracking settings%5$s.', 'pinterest-for-woocommerce' ),
+				implode( ', ', $third_party_tags ),
+				sprintf( '<a href="%s" target="_blank">', esc_url( admin_url( 'plugins.php' ) ) ),
+				'</a>',
+				sprintf( '<a href="%s" target="_blank">', esc_url( wc_admin_url( '&path=/pinterest/settings' ) ) ),
+				'</a>',
```

Comment:
I think it's better to interpolate in one place. 
Currently, we interpolate messages, URLs, `a`nchors here in PHP, then interpolate message within `<strong>` and `<span>` in JS. 

I'd rather interpolate everything in JS. 
Then we get:
- Entire interpolation logic in one place, to make it easier to change and maintain in future.
- Security of not stamping any HTML from external sources.
- Flexibility of styling, as then we could control any additional classes, attributes, or HTMLElements/ReactComponents used to render the links.

Reply:
Sounds good, my comment here would be that we also need the interpolated warning message to add it to the feed state. We probably can move the `<strong>` and `<span>` interpolation to PHP too in order to avoid duplicated code.

cc @tomalec @JPry

Reply:
The warning message in the feed state should not contain the `<strong>` fragment tho, since that fragment is rendered by the 'status_label' [here](https://github.com/woocommerce/pinterest-for-woocommerce/blob/308b801e0111be8e6123562024c9a6e391026748/src/API/FeedState.php#L359)

Reply:
@alexiglesias31 If we need to make use of the output in both JS and PHP, then I agree that it makes sense to do the interpolation in PHP.

Reply:
> The warning message in the feed state should not contain the `<strong>` fragment tho, since that fragment is rendered by the 'status_label'

I'm still a bit confused. Does it mean we need this `<strong>` to be interpolated that `dangerously…` way or not https://github.com/woocommerce/pinterest-for-woocommerce/pull/495/files#r947690286

Reply:
@tomalec we can use `createInterpolateElement` as you suggested to interpolate the `<strong>Potential conflicting plugins detected.</strong>` tag and keep the rest of interpolations on PHP, since we are re-using the warning message on the feed state info too, and another prefix [is used there](https://github.com/woocommerce/pinterest-for-woocommerce/pull/495/files#diff-d7374912c385ac51b92ce25d055029bdb751eeb9f7e1cddd17015e182ee015f9R359).

Reply:
@tomalec another alternative is to use a combination of `createInterpolateElement` and `RawHTML`

```
{ createInterpolateElement(
	__(
		'<strong>Potential conflicting plugins detected.</strong> <warningMessage/>'
	),
	{
		strong: <strong />,
		warningMessage: (
			<RawHTML>
				{
					wcSettings.pinterest_for_woocommerce
						.conflictingTagsWarning
				}
			</RawHTML>
		),
	}
) }
```

Reply:
I think we could as well re-use the interpolation code on the client side. 
But that would require a bit of  change around `extra_info` rendering.
Plus, a warning message could be considered something different that "extra info" and have its own property to be stored and handled a bit differently (even though eventually rendered in the same spot)